tests/src/Kernel/PatternAssertions/LinksBlockAssertion.php

```diff
@@ -4,12 +4,13 @@
 
 namespace Drupal\Tests\oe_bootstrap_theme\Kernel\PatternAssertions;
 
+use Drupal\Tests\oe_bootstrap_theme\Kernel\Paragraphs\ParagraphsTestBase;
 use Symfony\Component\DomCrawler\Crawler;
 
 /**
  * Class for asserting links block paragraphs.
  */
-class LinksBlockAssertion {
+class LinksBlockAssertion extends ParagraphsTestBase {
```

Comment:
I missed this the first time, this should extend the BasePatternAssert, same for ListingAssertion